src/main/webapp/app/shared/modal/ModifyTherapyModal.tsx

```diff
@@ -43,9 +48,76 @@ function ModifyTherapyModal({
 }
 
 const ModifyTherapyModalContent = observer(
-  ({ treatmentUuid, treatmentName, drugList, onConfirm, onCancel, modifyTherapyModalStore }: IModifyTherapyModalProps) => {
+  ({
+    treatmentUuid,
+    treatmentName,
+    drugList,
+    currentTreatments,
+    onConfirm,
+    onCancel,
+    modifyTherapyModalStore,
+  }: IModifyTherapyModalProps) => {
     const disableDeleteTherapy = modifyTherapyModalStore.selectedTreatments.length < 2;
     const isEmptyTherapy = modifyTherapyModalStore.selectedTreatments.some(therapy => therapy.length === 0);
+    const isDuplicate = isDuplicateTherapy();
```

Comment:
What's the point of adding this when you already have alreadyExists?

Reply:
`isDuplicate` refers to when there are multiple instances of the same treatment within the modal (i.e. to dropdowns contain exactly the same drugs). `alreadyExists` refers to when the therapy as a whole is already present for the cancer type (i.e. there is already a collapsible corresponding to that therapy). I have changed the name of `isDuplicateTherapy` to `isDuplicateTreatment` to clear this up.